app/models/ability.rb

```diff
@@ -41,9 +41,12 @@ def initialize(user)
       can :change_admin, :administrator
       can :change_editor, :administrator
       can :change_basic, :administrator
+      can :sort, :administrator
```

Comment:
I think these abilities are not used (sort:administrator and edit:administrator) since the admin ability to modify characteristics is captured by `can :manage, [... Characteristic ...]` in line 52 below.